force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +116,21 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if (a == null) {
+            return null;
+        }
+        Integer b = a / 2;
```

Comment:
There is a math class that will help with this, but you managed to figure it out using just integer division.